backend-docker/jmap.properties

```diff
@@ -0,0 +1 @@
+url.prefix=https://50e9-2402-9d80-85a-fe80-805b-e215-ab33-3def.ngrok-free.app
```

Comment:
what is this domain? is it static?

Reply:
It will change everytime the test workflow is run, so no, it is not static.

Reply:
Should change to `https://exmple.com`

Reply:
Done